soc/nordic/common/uicr/gen_uicr/Kconfig

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+menu "UICR generator options"
+
+config UICR_SAMPLE_GENERATE_PERIPHCONF
```

Comment:
What does the sample in the name stand for?

Reply:
It is trying to communicate to the reader that this Kconfig option is specific to this gen_uicr "sample".

As opposed to most other Kconfigs in ~/ncs/zephyr/soc, which are read by all zephyr images.

Maybe

GEN_UICR_SAMPLE_GENERATE_PERIPHCONF

Would be clearer?

Or do you have another idea?

Reply:
sample in the sense of example? That's what I assume and find confusing, this shouldn't be example code right?
Or in sample the sense of this one unit? image or target? 

Wouldn't `GEN_UICR_GENERATE_PERIPHCONF` be enough? Or would that be confusing because it only applies to the uicr target and not actually be set in the app or radio build?

Reply:
fixed